client/src/features/image.js

```diff
@@ -0,0 +1,26 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+const initialStateValue = {
+  ocrText: [],
+  imageUrl: [],
+  code: [],
+}
+
+const imageSlice = createSlice({
+  name: 'image',
+  initialState: { value: initialStateValue },
+  reducers: {
+    setOcrText: (state, action) => {
+      state.value.ocrText.push(action.payload)
+    },
+    setImageUrl: (state, action) => {
+      state.value.imageUrl.push(action.payload)
+    },
+    setCode: (state, action) => {
+      state.value.code = action.payload
+    },
```

Comment:
For better readability:
```suggestion
    addImage: (state, action) => {
      state.value.ocrText.push(action.payload.text)
      state.value.imageUrl.push(action.payload.imageUrl)
    },
    updateCode: (state, action) => {
      state.value.code = action.payload
    },
```
Modify other related parts accordingly.

Reply:
I modified the reducer and related parts